independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,716 @@
+                  "title": "Total results"
+                },
+                "resultsFrom": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "resultsFrom",
+                  "title": "Results form"
+                },
+                "previousResultsUri": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Previous results URL"
+                },
+                "nextResultsUri": {
+                  "type": "string",
+                  "description": "nextResultsUri",
+                  "title": "Next results URL"
+                },
+                "exceptionNoticeUri": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Exception notice URL"
+                },
+                "results": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "businessId": {
+                        "type": "string",
+                        "description": "",
+                        "title": "Business Id"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "",
+                        "title": "Company name"
+                      },
+                      "registrationDate": {
+                        "type": "string",
+                        "description": "",
+                        "title": "Registration date"
+                      },
+                      "companyForm": {
+                        "type": "string",
+                        "description": "",
+                        "title": "Company form"
+                      },
+                      "detailsUri": {
+                        "type": "string",
+                        "description": "",
+                        "title": "Details URL"
+                      },
+                      "liquidations": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "version": {
+                              "type": "integer",
+                              "format": "int32",
+                              "description": "version"
+                            },
+                            "name": {
+                              "type": "string",
+                              "description": "name"
+                            },
+                            "type": {
+                              "type": "string",
+                              "description": "type"
+                            },
+                            "registrationDate": {
+                              "type": "string",
+                              "description": "registrationDate"
```

Comment:
@tpertila All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)